### ISSUE TITLE
There should be a header for toggle switch in InkCanvas.

### ISSUE THREAD

**Issue by YuliKl** [microsoft/Xaml-Controls-Gallery#167]
**Describe the bug**

**Pre-requisite:** 
Turn on Narrator
Set Narrator in Item Mode.

**Steps to reproduce the bug**

1. Launch "XAML Controls Gallery" application.
2. Navigate to InkCanvas in all controls and activate it.
3. Navigate to Toggle switch->Toggle ON/OFF the switch and observe narrator reading. 

**Expected behavior**

Toggle switch should have a name property as "shape to draw"

**Actual behavior**

Name is having but that is representing as "rectangle/circle", it is not conveyed the purpose of the toggle switch.

User Impact: User will not be able to tell in which shape stroke is available as name of the toggle switch is missing.

**Screenshots**

![image](https://user-images.githubusercontent.com/13142665/63800871-ee145a80-c8c3-11e9-9bbc-c32f35fb146d.png)

**Version Info**

Gallery version (found on Settings page): 

Version Info : 1.2.6.0

Windows 10 version:
- [x] Insider Build (xxxxx)
- [ ] May 2019 Update (18362)
- [ ] October 2018 Update (17763)
- [ ] April 2018 Update (17134)
- [ ] Fall Creators Update (16299)
- [ ] Creators Update (15063)
- [ ] Anniversary Update (14393)

Device form factor:
- [x] Desktop
- [ ] Mobile
- [ ] Xbox
- [ ] Surface Hub
- [ ] IoT

**Additional context**

Copied from [internal bug](https://microsoft.visualstudio.com/OS/_workitems/edit/20984097)
